defaults/main.yml

```diff
@@ -19,11 +19,9 @@ rabbitmq_config_file: etc/rabbitmq/rabbitmq.config.j2
 rabbitmq_config_env_file: etc/rabbitmq/rabbitmq-env.conf.j2
 rabbitmq_env_config: {}
 
-# rabbitmq_debian_repo: deb http://www.rabbitmq.com/debian/ testing main
 #other repos
-rabbitmq_debian_repo: "deb https://dl.bintray.com/rabbitmq/debian {{ ansible_distribution_release }} main #bintray"
-# rabbitmq_debian_repo_key: https://www.rabbitmq.com/rabbitmq-release-signing-key.asc
-rabbitmq_debian_repo_key: https://bintray.com/user/downloadSubjectPublicKey?username=rabbitmq
+rabbitmq_debian_repo: "deb https://packagecloud.io/rabbitmq/rabbitmq-server/ubuntu/ xenial main"
```

Comment:
Not a blocker since the underlaying image is `xenial`, however should we use `{{ ansible_distribution_release }}` for futureproofing?